Include/object.h

```diff
@@ -261,6 +261,22 @@ _Py_ThreadId(void)
     __asm__ ("mrs %0, tpidrro_el0" : "=r" (tid));
 #elif defined(__aarch64__)
     __asm__ ("mrs %0, tpidr_el0" : "=r" (tid));
+#elif defined(__powerpc64__)
+    #if defined(__clang__) && __has_builtin(__builtin_thread_pointer)
```

Comment:
Is it worth it to use `_Py__has_builtin()` to support old Clang versions? Or does Clang support it since the very first Clang version?

Reply:
It worth to using it :) Updated!